### PR TITLE
Add cell.error field for error messages

### PR DESCRIPTION
**Pull request by zengchu2**
Attempt to address issue #525 .

Enable returning error messages to users when typing `cell.error` and return objects when typing `cell.value`

Test As Follows:
value
error

### CODE REVIEW

File: main/src/com/google/refine/model/Cell.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
     final public Serializable   value;
     @JsonIgnore
     final public Recon          recon;
-    
+
```
Comment (kushthedude) [inline main/src/com/google/refine/model/Cell.java]: unwanted change here!

**Comment by zengchu2** [comment]
Sure, no problem!

**Comment by wetneb** [comment]
Nice, thank you! Currently, unit tests for the `Cell` class are gathered in `CellTests`, do you think you could add your test there instead of creating a dedicated class for it?
Also, you should not need to use `createCSVProject` for this: just creating a `Cell` with an `EvalError` as value, and checking that getting the `error` field returns the corresponding error message should do.

**Comment by zengchu2** [comment]
Thanks for the quick suggestion, I will address this issue immediately!

**Pull request merged**